iphone/Maps/Bookmarks/Categories/BMCViewModel/BMCDefaultViewModel.swift

```diff
@@ -141,6 +142,10 @@ extension BMCDefaultViewModel {
     manager.shareAllCategories(completion: handler)
   }
 
+  func importCategories(from urls: [URL]) {
+    urls.forEach(manager.loadBookmarkFile(_:))
```

Comment:
To properly pass multiple files into the core, and then to load them faster, and to properly generate one combined notification, `BookmarkManager::LoadBookmarks` can be refactored. The part that scans the directory and gets a collection of files may stay the same, and the part that loads a collection of files can be shared between the existing and new implementations.

Note that files are loaded on a separate thread via `GetPlatform().RunTask(Platform::Thread::File, [this]` to avoid blocking the UI.

Reply:
I think that we shouldn't refactor the LoadBookmarks because it s the initial loading method that should not be called twice in the app (as we discussed earlier) but make a new method that will handle multiple URLs at once.

Reply:
file import is limited by one